### PR TITLE
Feedback changes

### PR DESCRIPTION
**Pull request by Abdulrehman53**
Implemented changes suggested

### CODE REVIEW

File: lib/utils/drawer_homescreen.dart [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 import 'package:ofypets_mobile_app/screens/favorites.dart';
+import 'package:ofypets_mobile_app/screens/order_history.dart';
+import 'package:ofypets_mobile_app/screens/retun_policy.dart';
+import 'package:ofypets_mobile_app/utils/constants.dart';
+import 'package:scoped_model/scoped_model.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:url_launcher/url_launcher.dart';
 
 class HomeDrawer extends StatefulWidget {
+  int favCount = 2;
```
Comment (sakinaboriwala) [inline lib/utils/drawer_homescreen.dart]: Please comment the hardcoded value.

File: lib/utils/drawer_homescreen.dart [hunk 2]
```diff
@@ -268,32 +318,34 @@ class _HomeDrawer extends State<HomeDrawer> {
...
+    throw 'Could not launch $uri';
+  }
+}
+
+_callMe(String phone) async {
+  // Android
+  final uri = 'tel:$phone';
+  if (await canLaunch(uri)) {
+    await launch(uri);
+  } else {
```
Comment (sakinaboriwala) [inline lib/utils/drawer_homescreen.dart:389]: As the uri is same in if & else, so please remove unnecessary check.

**Review by sakinaboriwala — CHANGES_REQUESTED** [review]
Looks good, just some minor changes.

**Review by sakinaboriwala — APPROVED** [review]
LGTM.

**Review by gopalshimpi — COMMENTED** [review]
LGTM 👍

**Pull request merged**